Scripts/install.sh

```diff
@@ -4,13 +4,14 @@ set -euo pipefail
 
 DOWNLOAD_URI=https://github.com/alcatraz/Alcatraz/releases/download/1.1.15/Alcatraz.tar.gz
 PLUGINS_DIR="${HOME}/Library/Application Support/Developer/Shared/Xcode/Plug-ins"
+XCODE_APP_PATH="$(xcode-select -p | sed s/\.app.*/.app/)"
 XCODE_VERSION="$(xcrun xcodebuild -version | head -n1 | awk '{ print $2 }')"
 PLIST_PLUGINS_KEY="DVTPlugInManagerNonApplePlugIns-Xcode-${XCODE_VERSION}"
 BUNDLE_ID="com.mneorr.Alcatraz"
 TMP_FILE="$(mktemp -t ${BUNDLE_ID})"
 
 # Remove Alcatraz from Xcode's skipped plugins list if needed
-defaults read -app Xcode "$PLIST_PLUGINS_KEY" > "$TMP_FILE" && {
+defaults read -app "$XCODE_APP_PATH" "$PLIST_PLUGINS_KEY" > "$TMP_FILE" 2> /dev/null && {
```

Comment:
Instead of filtering out all errors here, we could instead inverse grep for `Couldn't find an application` and `does not exist` , so other errors would still bubble up. The option you need is [`grep -v`](http://linux.die.net/man/1/grep)